match/lib/match/runner.rb

```diff
@@ -133,13 +136,47 @@ def update_optional_values_depending_on_storage_type(params)
       end
     end
 
-    def fetch_certificate(params: nil, working_directory: nil, specific_cert_type: nil)
```

Comment:
so working_directory was ignored here since 9d199278d3963413b916434b347cccc42c436b05. Nice finding/cleanup.

Reply:
Maybe [RuboCop::Cop::Lint::UnusedMethodArgument](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/UnusedMethodArgument) rule will help a little with such things?

Reply:
<img width="1093" alt="image" src="https://github.com/fastlane/fastlane/assets/24282/e958af75-f3bc-4c11-9ec9-b508cf467932">
Yup. I will have to check if we need to refresh our rubocop rules.